[EDU-1664] - Chat Online status example (#2302)
ably/docs: examples/chat-online-status/page.md

# Online status for chat applications

> Use online status to enable users to view the users currently online and their personally set statuses within a chat application.

Reviewer: ```suggestion
Online statuses enable users to view who else is online and ready to chat. 
```

> With online status users can see who is currently online, and set their own status messages such as "Online" or "Away". This adds a layer of context and personalization to the chat experience.

Reviewer: ```suggestion
Use online status to display which users are currently online in a chat room. Users can also manually update their status, such as to set themselves as 'Away'. 
```

> Online status adds value to various applications. It enables users to quickly understand the availability of others and communicate more effectively. For example, in a team collaboration app, users can see who's actively working and set their status to "In a meeting" when they're unavailable, improving overall communication efficiency.

Reviewer: ```suggestion
Online statuses enable users to quickly understand the availability of others to communicate more effectively. For example, in a team chat room, users can quickly see who is available to collaborate with, and who is currently busy, in a meeting, or away from their desks.
```

> Online status is implemented using [Ably Chat](https://ably.com/docs/products/chat). The Chat SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting chat features. It is built on top of Ably's core platform, and so it provides the same performance guarantees and scaling potential.

Reviewer: ```suggestion
Online statuses are implemented using [Ably Chat](https://ably.com/docs/products/chat). The Chat SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting chat features. It is built on top of Ably's core platform, and so it provides the same performance guarantees and scaling potential.
```
